src/frontends/onnx/frontend/src/frontend.cpp

```diff
@@ -159,15 +159,6 @@ void FrontEnd::add_extension(const std::shared_ptr<ov::Extension>& extension) {
     } else if (const auto& so_ext = std::dynamic_pointer_cast<ov::detail::SOExtension>(extension)) {
         add_extension(so_ext->extension());
         m_other_extensions.push_back(so_ext);
-    } else if (auto common_conv_ext = std::dynamic_pointer_cast<ov::frontend::ConversionExtension>(extension)) {
```

Comment:
(no related to the root problem) @itikhono Is it not duplicate to the branch started in line 162?

Reply:
it's not a duplicate
this branch for ov::frontend::ConversionExtension
the next is for ov::frontend::onnx::ConversionExtension